tests/test_provider.py

```diff
@@ -176,26 +175,6 @@ def test_contract_revert_custom_exception(owner, get_contract_type, accounts):
     assert err.value.inputs == {"addr": accounts[7].address, "counter": 123}
 
 
-def test_contract_dev_message(owner, get_contract_type):
```

Comment:
unfortunately, we can longer run this test without installing `ape-vyper`, maybe we will find a way to have a less integration-y test